### PR TITLE
Added keywords for Java and Python language definitions

### PR DESCRIPTION
**Pull request by shakeelsamsu**
Enhancing the growing libraries of language definitions.

Java.yaml: Add keywords
Python.yaml: Add keywords

Adds keywords to two language definitions.

(Continues https://github.com/coala/coAST/pull/18 )

### CODE REVIEW

File: data/Language/Java.yaml [hunk 1]
```diff
@@ -12,3 +12,52 @@ delimiters:
...
+  - switch
+  - synchronized
+  - this
+  - throws
+  - transient
+  - 'true'
+  - try
+  - void
+  - volatile
+  - while
```
[the remark this answers is not in the document]
Comment (shakeelsamsu) [reply data/Language/Java.yaml]: Oh ok. Is there some sort of significance behind the EOL or is it simply a convention?

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/19/commits/84e9ecbf9979fb45ceeed1a3692519c949ed421d

**Comment by jayvdb** [comment]
ack 84e9ecb

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**